docs/messaging/sms/postman.md

```diff
@@ -0,0 +1,75 @@
+# Using Postman to Test Enhanced Business SMS APIs
+
+For easy testing using [Postman](https://www.getpostman.com/), RingCentral provides a Postman 2.0 Collection for Enhanced Business SMS. If you use Postman, the we recommend using the collection as it provides authorization handling using Postman variables and environments as recommended by Postman.
+
+The collection is available here:
+
+* [Postman 2.1 Collection](../specs/ringcentral_sms_rest_apis.postman_collection.json)
```

Comment:
Why don't you link to Github directly? Or embed the spec? If the JSON file is placed within code_samples, then the CI/CD system will monitor for syntax errors.

Reply:
Let's do that in second iteration or when we have postman for entire MVP APIs. The final home of this spec would be in specs folder. I see Anton has already generated it and it's sitting in there, but needs QA and changes. For now, lets keep it here or may be I'll link to Github in next iteration. (like done in Engage Voice/Digital)